airbyte-integrations/connectors/source-google-sheets/source_google_sheets/spec.yaml

```diff
@@ -18,7 +18,7 @@ connectionSpecification:
     row_batch_size:
       type: integer
       title: Row Batch Size
-      description: The number of rows fetched when making a Google Sheet API call. Defaults to 200.
+      description: The number of rows fetched when making a Google Sheet API call. Defaults to 200. You can increase this value according to your needs to avoid rate limits if your data is particularly wide.
```

Comment:
```suggestion
      description: The number of rows fetched when making a Google Sheet API call. Defaults to 200. You can decrease this value to avoid rate limits if your data is particularly wide.
```